tests/Testing/MockServerTestCase.php

```diff
@@ -56,27 +56,7 @@ protected function defaultUnpublishMessageJson($key): string
         return '{"key":"'.$key.'","action":"unpublish","eventTime":null,"properties":{},"payload":null}';
     }
 
-    protected function assertPublishPostRequest(
-        RequestInfo $request,
-        string $uri,
-        string $expectedBody,
-        array $headers = null
-    ): void {
-        $this->assertIngestionPostRequest($request, $uri, $expectedBody, $headers);
-        $this->assertEquals('application/json; charset=UTF-8', $request->getHeaders()['Content-Type']);
-    }
```

Comment:
Now, both removed methods are the same.
The only difference was present or missing header.
So - replaced usages with assertIngestionPostRequest

Reply:
...added the missing
`        $this->assertEquals('application/json; charset=UTF-8', $request->getHeaders()['Content-Type']);`
to that method (in next fixup commit)